tests/test_mq.py

```diff
@@ -145,3 +145,44 @@ def test_mq_config_tagging(self):
         self.assertEqual(
             tags,
             {'Env': 'Dev'})
+
+    def test_mq_message_broker_vpc_id_filter(self):
+        session_factory = self.replay_flight_data('test_message_broker_vpc_filter')
+        p = self.load_policy(
+            {
+                'name': 'test-message-broker-vpc-filter',
+                'resource': 'message-broker',
+                'filters': [
+                    {
+                        'type': 'vpc',
+                        'key': 'VpcId',
+                        'value': 'vpc-0598080981e0332f9',
+                        'op': 'eq'
+                    }
+                ],
+            },
+            session_factory=session_factory,
+            cache=True,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
Do you want to also assert on the message-broker resource id?

Reply:
Good idea -- I'll make this change.